### PR TITLE
Improve config decode error messages

### PR DESCRIPTION
**Pull request by alexander-demichev**

### CODE REVIEW

**Comment by alexander-demichev** [comment]
Closes https://github.com/fusor/cpma/issues/347

File: pkg/transform/crio_transform.go [hunk 1]
```diff
@@ -179,7 +179,7 @@ func (e CrioTransform) Extract() (Extraction, error) {
 
 	var extraction CrioExtraction
 	if _, err := toml.Decode(string(content), &extraction); err != nil {
-		return nil, errors.Wrap(err, "Failed to decode crio, see error")
+		return nil, errors.Wrap(err, "Failed to decode crio. Check ssh user permissions and file content, see error")
```
Comment (gildub) [inline pkg/transform/crio_transform.go]: While at it, what about using `CRI-O` instead?

File: pkg/decode/decode.go [hunk 2]
```diff
@@ -18,7 +18,7 @@ func MasterConfig(content []byte) (*legacyconfigv1.MasterConfig, error) {
 
 	serializer := k8sjson.NewYAMLSerializer(k8sjson.DefaultMetaFactory, scheme.Scheme, scheme.Scheme)
 	if _, _, err := serializer.Decode(content, nil, masterConfig); err != nil {
-		return nil, errors.Wrap(err, "Failed to decode master config, see error")
+		return nil, errors.Wrap(err, "Failed to decode master config. Check ssh user permissions and file content, see error")
```
Comment (gildub) [inline pkg/decode/decode.go]: For this one and following `serializer.Decode` calls:
We should be able to detect a ssh error earlier -> connection/permission/etc/ issues and go that far in such cases.
So "Check file content" should suffice.

**Comment by gildub** [comment]
This one is not needed anymore as https://github.com/fusor/cpma/issues/347 has been addressed.

**Pull request closed without merging**